README.rst

```diff
@@ -1,5 +1,125 @@
-colcon-coverage-result
+colcon-lcov-result
 ==================
 
 An extension for `colcon-core <https://github.com/colcon/colcon-core>`_ to provide aggregate
-lcov results.
+coverage results using [LCOV](http://ltp.sourceforge.net/coverage/lcov.php).
+
+LCOV is a graphical front-end for GCC's coverage testing tool
+[gcov](https://gcc.gnu.org/onlinedocs/gcc/Gcov.html)), producing the following
+coverage metrics:
+
+- Statement coverage
+- Function coverage
+- Branch coverage
+
+For more information, see [this paper](http://ltp.sourceforge.net/documentation/technical_papers/gcov-ols2003.pdf)
+and [this Wikipedia page](https://en.wikipedia.org/wiki/Code_coverage).
+
+
+Usage
```

Comment:
Should we add installation instructions, or are we assuming this is installed in ADE?

Reply:
@jpsamper2009 thanks for the break down, I agree with you.